config/rbac/role.yaml

```diff
@@ -78,17 +78,6 @@ rules:
   - get
   - list
   - watch
-- apiGroups:
```

Comment:
don't we want to keep at least get,list,watch permissions for service accounts? IIRC we talked about examining them in users' namespaces

Reply:
Then I think we should add them back when we implement that feature.